pages/linux/smbmap.md

```diff
@@ -0,0 +1,36 @@
+# smbmap
+
+> SMB enumeration tool.
+> More information: <https://github.com/ShawnDEvans/smbmap>.
+
+- Display SMB shares and permissions on a host:
+
+`smbmap -u {{username}} -p {{password}} -H {{ip}}`
```

Comment:
Specifying a password like this will mean that the password ends up in the user's shell history. Is there another way we can pass the password to avoid this? Same goes for all the other examples.

Reply:
This tool is typically used during penetration tests. And indeed, you should carefully manage your history while using cleartext passwords in command lines. That's the way it is used in general.
But this is a very good remark ; looking at the source of the tool reveals that you can either use `-p {{password}}` (to specify the password in the command line) or `--prompt` (to prompt for a password using the `getpass.getpass()` function. I will adapt at least one example with the prompt option.

Reply:
I adapted one example. Good for you ?

Reply:
Heya! Ah, I see. That looks much better! Can this be reflected across the other examples please?

Reply:
Done.